Makefile

```diff
@@ -17,6 +19,12 @@ export CRI_DOCKERD_LDFLAGS=-ldflags "-s -w -buildid=${REVISION} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.PreRelease=${PRERELEASE} \
 	-X github.com/Mirantis/cri-dockerd/cmd/version.GitCommit=${REVISION}"
 
+all: cri-dockerd
+
+.PHONY: cri-dockerd
+cri-dockerd:
```

Comment:
Add a description for the `help` command. This or `help` should also be the default make target